pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -40,6 +40,8 @@ let
       dontInstall = true;
 
       postPatch = ''
+        patchShebangs $out/util/crossgcc/buildgcc
```

Comment:
That line was moved here ^

Reply:
The postPatch phase runs **after** the postFetch phase. So your change doesn't help when buildgcc is used during postFetch phase.

Reply:
That's okay, its shebang doesn't need to be patched anymore, because it is now passed as an argument to the shell up above, on line 29.